source/shaders/glyphquad.vert

```diff
@@ -0,0 +1,54 @@
+precision mediump float;
+precision lowp int;
+
+@import ./facade.vert;
+
+#if __VERSION__ == 100
+#extension GL_EXT_draw_buffers : enable
```

Comment:
use indentation of 4 spaces in this case, hard to read (this applies to the whole` #if #else #endif ` block)